pkg/gpu/context.go

```diff
@@ -46,20 +46,23 @@ func getSystemContext(nvmlLib nvml.Interface) (*systemContext, error) {
 	return ctx, nil
 }
 
-func (ctx *systemContext) queryDevices() error {
```

Comment:
I wouldn't rename this function, as in other PRs we'll need to get more data for each device (e.g., the compute architecture version) in this function.

Reply:
changed to `fillDeviceInfo`